minigotchi-ESP32/config.cpp

```diff
@@ -17,8 +18,8 @@ bool Config::deauth = true;
 bool Config::advertise = true;
 
 // screen configuration
-bool Config::display = false;
-std::string Config::screen = "";
+bool Config::display = false;  // Change to true if using a display
+std::string Config::screen = "";  // Define SSD1306, WEMOS_OLED_SHIELD or CYD depending on display used
```

Comment:
don't worry about comments, the install guide should cover that

Reply:
Removed.